python/cuml/test/test_train_test_split.py

```diff
@@ -57,6 +57,46 @@ def test_split_dataframe(train_size, shuffle):
     assert all(out)
 
 
+@pytest.mark.parametrize("train_size", [0.2, 0.6, 0.8])
+@pytest.mark.parametrize("shuffle", [True, False])
```

Comment:
Do we need the full matrix of all of these parameters? Seems to me that `y_type` might be the only one that is relevant for this particular test, no?

Reply:
That's true, I was mimicking the earlier test, but even without that it checks the PR. Removed it now